### DISCUSSION TITLE
The backup process stops after the first failure

### DISCUSSION THREAD

**Discussion by dejagersh** [spatie/laravel-backup#1708] [Ideas]
I use `spatie/laravel-backup` to back up a bunch of databases: around seven, within a single Laravel app that is. When the first DB fails to be backed up, the whole command fails. It would be great if `backup:run` would just continue with DB 2 to 7 and save those 6 backups, then notify me of a failure of the first DB.

Because now, I get no backups of _any_ DB, even though there might only be 1 DB e.g. unreachable.

**Comment by angeljqv** [comment]
Back up individually

**Reply by dejagersh** [reply]
Elaborate please? By using 7 different `laravel-backup` installations? I use 1 Laravel app to backup many databases, also DBs used by non-Laravel apps. That's way more convenient than installing `laravel-backup` on all apps + that doesn't work for non-Laravel apps.

**Reply by angeljqv** [reply]
> By using 7 different laravel-backup installations?

😄😄😄😄 Please read documentation or discussions section, it's a basic config

[Backup Individually](https://github.com/search?q=repo%3Aspatie%2Flaravel-backup%20%2Fdb-name%2F&type=code)

**Reply by dejagersh** [reply]
Yea so I'd have to write some code that wraps `backup:run`, understood. It will send me 7 notifications and it will create 7 zip files, but it would solve the problem of "missing backups".

I don't think it's a great solution though. Hence my idea / feature request to change the behaviour of `backup:run`. If you don't think it's worth changing, that's fine.

**Reply by angeljqv** [reply]
Use [spatie/db-dumper](https://github.com/spatie/db-dumper), and zip the dbs, if one is missing send you a notification, works for non-laravel apps too
